src/components/pages/UpdateFarmer/UpdateFarmer.js

```diff
@@ -97,17 +94,40 @@ const UpdateFarmer = ({ location, history, appStateShouldUpdate, user }) => {
         newEntry.selected = [...newEntry.selected, value];
       }
     } else if (type === 'file') {
-      const imageFile = new FormData();
-      imageFile.append('file', files[0]);
-      imageFile.append(
-        'upload_preset',
-        process.env.REACT_APP_CLOUDINARY_PRESET
-      );
-      const imageUrl = await axios
-        .post(process.env.REACT_APP_CLOUDINARY_URL, imageFile)
-        .then(data => data.data.secure_url)
-        .catch(err => err);
-      newEntry.imageUrl = imageUrl;
+      // Render the image in the form's <img /> element
+      e.persist();
+      
+      if (files.length) {
+        const imageFile = new FormData();
+        imageFile.append('file', files[0]);
+        imageFile.append(
+          'upload_preset',
+          process.env.REACT_APP_CLOUDINARY_PRESET
+        );
+
+        try {
+          if (process.env.REACT_APP_CLOUDINARY_URL) {
+            const uploadResponseData = await axios.post(
```

Comment:
Same than in previous requests, an `imageUploadHandler` could be good.

Reply:
Will be addressed in another PR. 👍